eng/pipelines/templates/steps/retain-heap-dump-hprofs.yml

```diff
@@ -0,0 +1,17 @@
+steps:
+  - pwsh: |
+      New-Item $(Build.ArtifactStagingDirectory)/oom-hprofs -ItemType directory
+      foreach($hprofFile in (Get-ChildItem -Path . -Recurse -Filter *.hprof -File))
+      {
+        $fileFullName = $hprofFile.FullName
```

Comment:
looks like this full name isn't used.

Reply:
It's used two lines below in the Move-Item command 😃

Reply:
https://github.com/Azure/azure-sdk-for-java/pull/18392/files/036002bb2b81a394b1af1205ba1120d908fc07cb#diff-c0636fb18157703f77203332e4169a450ca3084e7a26a3285bceca46e3e19152R8